design/API.md

```diff
@@ -125,16 +125,16 @@ Here is an example Search Space JSON
 }
 ```
 
-## Stop experiment
-Stop a running experiment before the experiment has finished
+## Delete experiment
+Delete an experiment. Data/plots available for the experiment also gets deleted.
```

Comment:
Can you elaborate here, what happens to an ongoing experiment or only if the experiment has completed